classes/class-wc-connect-help-provider.php

```diff
@@ -145,6 +145,14 @@ protected function get_health_items() {
 					__( 'Please connect Jetpack to WordPress.com', 'woocommerce' ),
 					''
 				);
+			} else if ( Jetpack::is_staging_site() ) {
+				$health_item = $this->build_indicator(
+					'jetpack_indicator',
+					'notice',
+					'indicator-error',
+					__( 'This is a Jetpack staging site, some features may not work as expected', 'woocommerce' ),
```

Comment:
Well, for now everything works, except for our internal stats, so I don't know what else to put here. Is there anything that isn't supposed to work in Jetpack Staging?